hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancer.java

```diff
@@ -511,7 +511,7 @@ public MockCluster(int[][] regions) {
     }
 
     @Override
-    float getLocalityOfRegion(int region, int server) {
+    float getLocalityOfRegion(int region, int server, boolean onlyOnSsd) {
       // convert the locality percentage to a fraction
       return localities[region][server] / 100.0f;
     }
```

Comment:
Needs test to demonstrate this new attribute works? Has an effect?

Reply:
Added little test for the class of ServerSsdLocalityCostFunction.
BTW, the logic of getLocalityOfRegion is simple, so i think maybe it is no need to test it independent.
WDYT?